core/translate/expr.rs

```diff
@@ -1550,6 +1550,57 @@ pub fn translate_expr(
                             });
                             Ok(target_register)
                         }
+                        ScalarFunc::Replace => {
+                            let args = if let Some(args) = args {
+                                if !args.len() == 3 {
+                                    crate::bail_parse_error!(
+                                        "{} function with wrong number of arguments",
```

Comment:
could we improve on sqlite's error message here and say `function {}() requires exactly 3 arguments` or similar?

Reply:
I've changed the message with the latest commit.